docs-src/content/functions/data.yml

```
@@ -312,7 +312,7 @@ funcs:
         description: the (single-character!) field delimiter, defaults to `","`
       - name: header
         required: false
-        description: comma-separated list of column names, set to `""` to get auto-named columns (A-Z), defaults to using the first line of `input`
+        description: character-separated list of column names using delim as separator, set to `""` to get auto-named columns (A-Z), defaults to using the first line of `input`
```

Just thinking about the wording - this seems a bit awkward. How about simply:

```suggestion
        description: list of column names separated by `delim`, set to `""` to get auto-named columns (A-Z), defaults to using the first line of `input`
```